board/app/controllers/user_controller.php

```diff
@@ -78,15 +75,132 @@ public function login()
         $this->render($page); 
     }
         
-    public function logout()
-    {
+    public function logout() {
         session_destroy();
         redirect(url('user/login'));
     }
 
-    public function home()
-    {
+    /*
+    * Display user's name, username, and email
+    
+    public function profile() {
+
+        $user = User::getData();
+        $this->set(get_defined_vars());
+    }*/
+
+     /*
+* Display user's name, username, and email
+*/
+public function profile() {
+$user = User::getData($_SESSION['user_id']);
+$this->set(get_defined_vars());
+}
+
+
+    public function home(){
+
+    }
```

Comment:
Yes po Ma'am. I can't view home.php po without this function. :bow: :disappointed: